### ISSUE TITLE
Beobachtung in EKF umwandeln

### ISSUE THREAD

**Issue by rebeccakurz** [barbalex/apf2#585]
Könnte eine Beobachtung mit einem Befehl in eine Erfolgskontrolle (durch Freiwillige) umgewandelt werden? Bzw. könnte mit den Infos aus der Beobachtung (Beobachter, Anzahl Pflanzen, etc) eine Freiwilligen-Kontrolle erstellt werden?

**Comment by barbalex** [comment]
Jein.

Technisch: Mit ein paar anspruchsvollen Problemen: ja. 
- Anzahl Pflanzen sind prinzipiell nicht nutzbar, weil nicht alle Beobachtungen dieselbe Datenstruktur haben. Ev. müsste die AV bei diesem Vorgang angeben, in welchem Feld diese Angabe steckt.
- Die Namen der Beobachter sind zwar bekannt, aber nicht als Adressen verfügbar, wie die Beobachter von Kontrollen. Die AV müsste also beim Import auswählen, ob diese Person bereits als Adresse in apflora existiert (das geht nicht automatisch, weil Schreibweisen variieren). Und entweder die neue Person als Adresse erfassen. Oder die passende bestehende Adresse wählen. Oder die Namen von Beobachtern werden nicht mehr mit Adressen verknüpft und in reinen Text migriert. Was aufwändig ist und Folgen hat für die Auswertbarkeit der Daten.

Die wichtigere Frage ist: Macht das Sinn?

apflora erfasst und dokumentiert, was im Projekt geleistet wird. Externe Beobachtungen sind keine Leistung des Projekts aber werden natürlich für einen gesamheitlichen Überblick benutzt.

Aus ähnlichen Gründen gibt as dann ein Problem mit dem Beobachtungs-Daten-Kreislauf: Kontrollen werden als Beobachtungen                    an Info Flora exportiert. Das mit als Kontrollen importierten Beobachtungen zu verhindern wäre komplex und fehlerträchtig, wenn auch prinzipiell möglich (obs_id in Kontrollen speichern).

Fragt sich also einfach, ob es sinnvoll ist und sich der Aufwand lohnt.

Die Umsetzung bräuchte genaue Absprachen und mehrere Tage Aufwand.

**Comment by rebeccakurz** [comment]
Ich denke, wenn der Aufwand für eine Umsetzung so gross wäre, dann ist es einfacher, wenn die Artverantwortlichen aus den Beobachtungen von Hand eine Kontrolle erstellen, wenn sie das als sinnvoll erachten.

**Comment by barbalex** [comment]
Achtung: das Problem mit dem Austausch der Beobachtungen bleibt auch so

**Comment by rebeccakurz** [comment]
Ja, das habe ich mir auch überlegt. Der Wunsch kam von Anna Wunderli, ich werde sie sicher noch darüber informieren, damit sie sich dieses Problems auch bewusst ist.

**Comment by rebeccakurz** [comment]
Aufwand grösser als Nutzen
